index.js

```diff
@@ -980,6 +980,42 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] -> Maybe [a]
+  //.
+  //. Returns Just a list containing the first n elements of the given
+  //. list or Just the first n characters of a given string. If n exceeds
+  //. the length of the list or string, returns Nothing.
```

Comment:
Perhaps we could use the word "collection":

_Returns Just the first N elements of the given collection if N is less than or equal to the length of the collection; Nothing otherwise. Supports Array, String, and any other collection type which provides a `slice` method._